src/main.js

```diff
@@ -133,6 +133,7 @@ async function push() {
         core.info(`Label ${label} added to #${pull_number}.`);
         core.debug(JSON.stringify(labelsResponse.data));
     }
+    core.setOutput("pull-request", pull_number.toString());
```

Comment:
I think we're supposed to add this to the `action.yml` file. Check out the [documentation](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/metadata-syntax-for-github-actions#outputs).